### ISSUE TITLE
ACS commons Disptacher TTL functionality is not working when  we upgrade from version number1.10.2 to 2.5.0 due to .ttl file not created on dispatcher 

### ISSUE THREAD

**Issue by subramanya** [Adobe-Consulting-Services/acs-aem-commons#756]
On our environment we had ACS commons version 1.10.2 . We needed the dispatcher TTL functionality for our cached web pages. We upgraded the ACS commons bundle version to 2.5.0.
We are deploying ACS commons package as a dependency in our project's pom.xml file. when I verified on the AEM instance I found that .ttl file is not getting created on dispatcher file system for the files that should be invalidated by Dispatcher TTL functionality. 
I installed dispatcher ACS -commons 2.2.0 on a vanilla AEM instance and found the for the configured pages .ttl file is getting created on dispatcher (under document root ) for the configured files . When I upgraded ACS commons bundle to 2.5.0 the .ttl file was not created for the configured file on dispatcher and It broke the TTL functionality
Hence the issue is on ACS commons 2.5.0 version

**Comment by davidjgonzalez** [comment]
@subramanya 

1. What version of AEM are you running? Seems like you are doing an odd upgrade of 1.x -> 2.x on the same AEM instance?
2. Have you verified that the 2.x ACS Commons bundle is active (/system/console/bundles)
3. Have you verified that the Dispatcher TTL Filter Components are active (/system/console/components)
4. Have you verified that the Dispatcher TTL OSGi Configurations are properly configured to produced the expected results (/system/console/configMgr)

**Comment by bstopp** [comment]
I was able to verify that this works between 2.3 and 2.5. 

Make sure that:

- there's no Authorization header (or Dispactcher won't cache)
- There's no Cache-Control header; or the Filter will not process the request. 
  - This header should be blocked at the Dispatcher; but many implementations set allowedheaders => '*' which _will_ forward any cache Control header.

I used PostMan to send a request, w/o the Cache-Control header and everything worked as expected.
